week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,11 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+function percentage(numberOfMentors, numberOfStudents) {
+  var total = numberOfMentors + numberOfStudents;
+
+  var percentageStudents = Math.round((numberOfStudents * 100) / total);
+  var percentageMentors = Math.round((numberOfMentors * 100) / total);
+
+  console.log("Percentage Students: " + percentageStudents + "%");
```

Comment:
I like that you have included the percentage sign "%"

Reply:
Thank you for the feedback Tayo @Atayo! :smile: I appreciate it!